### PR TITLE
Updated README to reflect maintenance status

### PR DESCRIPTION
**Pull request by kaczmarczyck**
This PR is to be merged after the branches are reworked.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-![markdownlint](https://github.com/google/OpenSK/workflows/markdownlint/badge.svg?branch=stable)
-![pylint](https://github.com/google/OpenSK/workflows/pylint/badge.svg?branch=stable)
-![Cargo check](https://github.com/google/OpenSK/workflows/Cargo%20check/badge.svg?branch=stable)
-![Cargo format](https://github.com/google/OpenSK/workflows/Cargo%20format/badge.svg?branch=stable)
-[![Coverage Status](https://coveralls.io/repos/github/google/OpenSK/badge.svg?branch=stable)](https://coveralls.io/github/google/OpenSK?branch=stable)
-
-*News:*
-
-- 2023-08-24: [PQC paper reference](#Research)
+This branch is unmaintained. It implements the FIDO 2.0 version of OpenSK.
```
Comment (jmichelp) [inline README.md]: CTAP2.0

File: README.md [hunk 2]
```diff
@@ -76,28 +66,6 @@ Please check our [Troubleshooting and Debugging](docs/debugging.md) section if y
 have problems with the installation process or during development. To find out what
 else you can do with your OpenSK, see [Customization](docs/customization.md).
 
-## Research
```
Comment (jmichelp) [inline README.md:79]: What would be the landing branch?
For visibility I think it's better if we keep a Research or Publications section in the README for the default branch

Comment (kaczmarczyck) [reply README.md:79]: The landing branch will be 2.1, and I will merge this PR after it is properly set up.

File: README.md (hunk 1 above)
Comment (kaczmarczyck) [reply README.md]: Done

**Review by jmichelp — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,14 +1,9 @@
...
-![pylint](https://github.com/google/OpenSK/workflows/pylint/badge.svg?branch=stable)
-![Cargo check](https://github.com/google/OpenSK/workflows/Cargo%20check/badge.svg?branch=stable)
-![Cargo format](https://github.com/google/OpenSK/workflows/Cargo%20format/badge.svg?branch=stable)
-[![Coverage Status](https://coveralls.io/repos/github/google/OpenSK/badge.svg?branch=stable)](https://coveralls.io/github/google/OpenSK?branch=stable)
-
-*News:*
-
-- 2023-08-24: [PQC paper reference](#Research)
+This branch is unmaintained. It implements the CTAP 2.0 version of OpenSK.
+Please check out the latest versioned branch for maintained code.
```
Comment (ia0) [inline README.md]: What is the latest versioned branch? The default branch?

Comment (kaczmarczyck) [reply README.md]: Yeah, I can call it default branch?

Comment (ia0) [reply README.md]: That's the Github term yes. When you list branches on Github you see "default" next to it.

**Review by ia0 — APPROVED** [review]

**Review by jmichelp — APPROVED** [review]

**Pull request merged**